deploy/infrastructure/dependencies/terraform-commons-dss/helm.tf

```diff
@@ -91,6 +91,32 @@ resource "local_file" "helm_chart_values" {
     global = {
       cloudProvider = var.kubernetes_cloud_provider_name
     }
+
+    monitoring = {
+      enabled = false
+    },
+
+    prometheus = {
+      server = {
+        persistentVolume = {
+          storageClass = var.kubernetes_storage_class
+        }
+
+        global = {
+          external_labels = {
+            environment = "dev"
```

Comment:
Why is this hard-coded ?

Reply:
https://github.com/interuss/dss/issues/1329